### ISSUE TITLE
Experiment "module_variable_optional_attrs" is no longer available. 

### ISSUE THREAD

**Issue by fe-fo** [aws-ia/terraform-aws-vpc#91]
Hello!
Getting below error when trying to use the module from TF v1.3.2
[Github Issue](https://github.com/hashicorp/terraform/issues/31692)

```
module "vpc" {
  source  = "aws-ia/vpc/aws"
  version = "3.0.0"

  name       = "vpc"
  cidr_block = "10.0.0.0/20"
  az_count   = 3

  subnets = {
    public = {
      netmask                   = 24
      nat_gateway_configuration = "all_azs" # options: "single_az", "none"
    }

    private = {
      netmask                 = 24
      connect_to_public_natgw = true
    }
  }

  vpc_flow_logs = {
    log_destination_type = "cloud-watch-logs"
    retention_in_days    = 180
    traffic_type         = "REJECT"
  }
}
```
![image](https://user-images.githubusercontent.com/36716963/196568026-680fdfd1-25f1-42d7-a284-4d1fd5b928f8.png)

**Comment by drewmullen** [comment]
This will be fixed tomorrow. iirc @pablo19sc has a code change ready to resolve

Thank you for reporting

**Comment by inas17** [comment]
Same issue on Azurerm backend: │ Error: Experiment has concluded
│ 
│   on .terraform/modules/key-vault/versions.tf line 2, in terraform:
│    2:   experiments = [module_variable_optional_attrs]
│ 
│ Experiment "module_variable_optional_attrs" is no longer available. The final feature corresponding
│ to this experiment differs from the experimental form and is available in the Terraform language from
│ Terraform v1.3.0 onwards.

**Comment by pablo19sc** [comment]
Hi! We have released a new version of the VPC Module (v3.0.1) today that solves this issue. The terraform version required now is > 1.3.0
